lib/src/gateway/shard_runner.dart

```diff
@@ -465,6 +467,9 @@ Stream<dynamic> decompressTransport(Stream<List<int>> raw) {
   });
 }
 
+// See https://github.com/instantiations/es_compression/issues/52 for why the isNotEmpty check is needed.
+Stream<dynamic> decompressZStdTransport(Stream<List<int>> raw) => raw.transform(zstd.decoder).where((message) => message.isNotEmpty);
```

Comment:
is isNotEmpty still needed? https://github.com/instantiations/es_compression/issues/52#issuecomment-2336428316